foreign_cc/cmake.bzl

```diff
@@ -393,6 +393,14 @@ def _attrs():
             doc = "Arguments for the CMake install command",
             mandatory = False,
         ),
+        "set_file_prefix_map": attr.bool(
```

Comment:
Couldn't we just add this to `CC_EXTERNAL_RULE_ATTRIBUTES` rather than having to modify each of the individual rulesets?

Reply:
I've just checked this works. Obviously better to add to `CC_EXTERNAL_RULE_ATTRIBUTES`